booking-app/src/main/java/org/motechproject/bookingapp/service/impl/VisitBookingDetailsServiceImpl.java

```diff
@@ -74,4 +88,120 @@ public void createOrUpdate(Collection<VisitBookingDetails> visitBookingDetailsLi
             }
         }
     }
+
+    @Override
+    public List<VisitBookingDetailsDto> getVisitDtos(int page, int pageSize, String sortColumn, String sortDirection) {
```

Comment:
let's move this logic to a new servie called PrimeVaccinationScheduleService - this isn't striclty related to VisitBookingDetails